fix(CSS): update `minmax()` value `auto` (#27783)
mdn/content: files/en-us/web/css/minmax/index.md

- `min-content`
  - : Represents the largest min-content contribution of the grid items occupying the grid track.
- `auto`

>   - : As a _maximum_, identical to `max-content`. However, unlike `max-content`, allows expansion of the track by the `align-content` and `justify-content` property values like `normal` and `stretch`.\
>     As a _minimum_ it represents the largest minimum size (as specified by {{cssxref("min-width")}}/{{cssxref("min-height")}}) of the grid items occupying the grid track.

Reviewer: - Because we mention `min` and then `max`, in that order, I've suggested swapping the description to follow that order.
- Also, earlier in the prose, we're referring to the minimum and maximum as `min` and `max`, I've suggested using those terms for consistency. (_A function taking two parameters, `min` and `max`._)

```suggestion
  - : As `min`, it represents the largest minimum size (as specified by {{cssxref("min-width")}}/{{cssxref("min-height")}}) of the grid items occupying the grid track.
    As `max`, it is identical to `max-content`. However, unlike `max-content`, it allows expansion of the track by the {{cssxref("align-content")}} and {{cssxref("justify-content")}} property values like `normal` and `stretch`.
```
